Update icecube info (#988)
nasa-gcn/gcn.nasa.gov: app/routes/missions/icecube.mdx

| [`ICECUBE ASTROTRACK GOLD/BRONZE`](http://gcn.gsfc.nasa.gov/doc/IceCube_High_Energy_Neutrino_Track_Alerts_v2.pdf) | 0.5–1 minutes | High-energy single neutrino track event alerts           | [Alerts](https://gcn.gsfc.nasa.gov/amon_icecube_gold_bronze_events.html) |
| [`AMON ICECUBE CASCADE`](http://gcn.gsfc.nasa.gov/doc/High_Energy_Neutrino_Cascade_Alerts.pdf)                    | 0.5–1 minutes | High-energy neutrino cascade event alerts                | [Alerts](https://gcn.gsfc.nasa.gov/amon_icecube_cascade_events.html)     |
| [`AMON NU EM COINC`](https://gcn.gsfc.nasa.gov/doc/gamma_nu.pdf)                                                  | 7 hours       | AMON Coincidence alerts of IceCube+HAWC or ANTARES+Fermi | [Alerts](https://gcn.gsfc.nasa.gov/amon_nu_em_coinc_events.html)         |

</div>

## GCN Notice Types in GCN Over Kafka

**_IceCube LVK Alert Nu Track Search_**\
**topic = gcn.notices.icecube.lvk_nu_track_search**\
IceCube performs realtime searches for coincident neutrino signals with all

> [LVK Gravitational alerts](https://emfollow.docs.ligo.org/userguide/), using

Reviewer: ```suggestion
[LIGO/Virgo/KAGRA gravitational-wave alerts](https://emfollow.docs.ligo.org/userguide/), using
```

a realtime muon neutrino track event selection[^3] and the sky maps from gravitational

> wave detectors. These low-latency joint searches identify neutrino signals within a +/-500 second time

Reviewer: ```suggestion
wave detectors. These low-latency joint searches identify neutrino signals within a ±500 second time
```

window surrounding the LVK alert time, and aim to identify multi-messenger transient
sources and seed electromagnetic followup up observations[^4] [^5].

These searches are performed as quickly as possible, as soon as the realtime neutrino information for the 1000 second window
are available, and are repeated for each update
to the LVK alert information. The results from each repeated search will sent as a GCN notice,
including identifying information to the corresponding LVK alert version.

For high-significance LVK alerts, two hypothesis tests
are conducted. The first search is a maximum likelihood analysis which searches for a
generic point-like neutrino source coincident with the given GW sky map. The
second uses a Bayesian approach[^6] to quantify the joint GW + neutrino event significance, which
assumes a binary merger scenario and accounts for known astrophysical priors, such as GW source
distance, in the significance estimate. For low-significance LVK alerts, only the Bayesian search
is performed. All searches are reported via GCN Notices over Kafka, and any coincident
observation with a observed p-value less than 1\% will also

> be sent as a [GCN Circular](https://gcn.nasa.gov/circulars).

Reviewer: ```suggestion
be sent as a [GCN Circular](/circulars).
```

The GCN Notice will highlight:

> - Overall search results p-values for the generic and Bayesian searches ("pval_generic" and "pval_bayesian")
> - If the analysis p-value from either search is less than 10%, the number ("n_events_coincident") and per-event information for coincident neutrino events (for events with a per-event p-value of less than 10%) are also provided:
>   - "event_dt" - relative timing of neutrino to GW alert (sec)
>   - "ra", "dec" - reconstructed neutrino event direction (deg., J200),
>   - "ra_uncertainty" - circular direction error (deg.) at 90\% confidence.
>   - "event_pval_generic", "event_pval_bayesian" - per-event p-values for each search. In the case of multiple coincidences, can be used to determine relative importance of each event to search results.
> - "most_probable_direction" - RA/DEC (deg., J2000) for the most likely direction from the generic neutrino source coincidence search.
> - "neutrino_flux_sensitivity_range" - Time integrated flux sensitivity range (min, max) [GeV cm^-2] and energy sensitivity range (lower, upper) [GeV], assuming an E^-2 neutrino spectrum (E^2 dN/dE) found within the points in the 90% region of GW map localization"

Reviewer: ```suggestion
- Overall search results p-values for the generic and Bayesian searches (`pval_generic` and `pval_bayesian`)
- If the analysis p-value from either search is less than 10%, the number (`n_events_coincident`) and per-event information for coincident neutrino events (for events with a per-event p-value of less than 10%) are also provided:
  - `event_dt` - relative timing of neutrino to GW alert (sec)
  - `ra`, `dec` - reconstructed neutrino event direction (deg., J200),
  - `ra_uncertainty` - circular direction error (deg.) at 90% confidence.
  - `event_pval_generic`, `event_pval_bayesian` - per-event p-values for each search. In the case of multiple coincidences, can be used to determine relative importance of each event to search results.
- `most_probable_direction` - RA/DEC (deg., J2000) for the most likely direction from the generic neutrino source coincidence search.
- `neutrino_flux_sensitivity_range` - Time integrated flux sensitivity range (min, max) [GeV cm<sup>-2</sup>] and energy sensitivity range (lower, upper) [GeV], assuming an E<sup>-2</sup> neutrino spectrum (E<sup>2</sup> dN/dE) found within the points in the 90% region of GW map localization
```

> The [GCN schema](https://gcn.nasa.gov/schema/gcn/notices/icecube/LvkNuTrackSearch.schema.json) and

Reviewer: ```suggestion
The [GCN schema](/schema/gcn/notices/icecube/LvkNuTrackSearch.schema.json) and
```

example [JSON message](https://raw.githubusercontent.com/nasa-gcn/gcn-schema/main/gcn/notices/icecube/LvkNuTrackSearch.example.json)
are available.

Please note: The reported p-values here do not account for any trials correction (multiple hypotheses
testing). The false alarm rate of these coincidences can be obtained by multiplying the p-
values with their corresponding GW trigger rates. The triggering event rate for the
LLAMA pipeline which uses confident and subthreshold GW triggers is about 16/day (8
pipelines with 2/day FAR for each). The triggering event rate for the UML pipeline which
only uses the confident GW triggers is the expected astrophysical GW detection rate

> (see [LVK Public Alerts guide](https://emfollow.docs.ligo.org/userguide/changes.html)).

Reviewer: ```suggestion
(see [LIGO/Virgo/KAGRA Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/changes.html)).
```

## Common GCN Circular Types

<div className="overflow-table">

| Type                                                    | Latency | Example                                                     |
| ------------------------------------------------------- | ------- | ----------------------------------------------------------- |

> | Identification of high energy neutrino candidate        | ~hours  | [IceCube-220524A](https://gcn.nasa.gov/circulars/32102)     |
> | Follow up of high energy neutrino candidate             | ~days   | [IceCube-220524A](https://gcn.nasa.gov/circulars/32114)     |
> | Followup of other transient events for neutrino signals | ~days   | [ GRB 230307A](https://gcn.nasa.gov/circulars/33430)        |
> | Follow up of gravitational wave trigger                 | ~hours  | [LIGO/Virgo S200129m](https://gcn.nasa.gov/circulars/26927) |

Reviewer: ```suggestion
| Identification of high energy neutrino candidate        | ~hours  | [IceCube-220524A](/circulars/32102)     |
| Follow up of high energy neutrino candidate             | ~days   | [IceCube-220524A](/circulars/32114)     |
| Followup of other transient events for neutrino signals | ~days   | [ GRB 230307A](/circulars/33430)        |
| Follow up of gravitational wave trigger                 | ~hours  | [LIGO/Virgo S200129m](/circulars/26927) |
```

import logo from '~/img/icecube-logo.png'

# IceCube Neutrino Observatory

<img
  src={logo}
  alt="IceCube logo"
  align="right"
  width="150"

>   height="135"

Reviewer: These numbers don't match the dimensions or the aspect ratio of the image file.

Author: fixed

  className="height-auto"
/>

**Construction Completion Date:** December 2010

**End of Operations:** No specific requirement

**Data Archives:**\
https://dataverse.harvard.edu/dataverse/icecube \
https://icecube.wisc.edu/science/data-releases/ \

> https://heasarc.gsfc.nasa.gov/W3Browse/icecube/icecubepsc.html

Reviewer: Could we turn this into a bulleted list with actual descriptive link text, rather than just bare URLs?

The [IceCube Neutrino Observatory](https://icecube.wisc.edu/) is a cubic-kilometer Cherenkov particle detector
deployed in the Antarctic ice beneath the Amundsen-Scott South Pole Station. It consists of 86 strings of
photo-detectors, extending to a depth of about 2,500 meters below the glacier's surface and
instrumenting a cubic-kilometer of ice. The Digital Optical Module photo-detectors detect the light
produced by relativistic charged particles produced by neutrino interactions in or near the instrumented
volume of ice.

> IceCube is sensitive to neutrinos from all directions (all-sky). Neutrino signatures can be broadly

Reviewer: IMHO, this clarification is not necessary:

```suggestion
IceCube is sensitive to neutrinos from all directions. Neutrino signatures can be broadly
```

Author: removed

> classified as track-like, produced by the elongated (~km) emission from secondary muons, or as

Reviewer: The way that this sentence is worded, it can easily be misread as stating that the neutrinos are produced by secondary muons, whereas (if I understand correctly) it is the other way around, the muons are produced by neutrinos interacting with the ice.

Consider instead: "As neutrinos pass through the ice, they can leave track signatures in the IceCube detector array when they produce secondary muons or compact signatures when they produce secondary electrons or hadrons."

Author: Thanks, inspired by yours, now reads:  
As neutrinos pass through the ice, their interactions can 
leave track signatures (~km in length) in the IceCube detector array when they produce secondary muons or compact 
signatures (cascades of ~m in extent) when they produce secondary electrons or hadrons.

cascade events, compact events (~m) produced electrons or hadronic secondary particles.
Track events can be reconstructed with an uncertainty of less than 1 degree, while cascade events
have higher signal purity. IceCube triggers on signals for neutrino energies greater than ~10 GeV

> (10<sup>10</sup> eV) , and can identify likely astrophysical neutrino events for neutrino

Reviewer: ```suggestion
(10<sup>10</sup> eV) and can identify likely astrophysical neutrino events for neutrino
```

Author: fixed

> energies greater than ~50 TeV (5 x 10<sup>13</sup> eV) and issue alerts to the community.

Reviewer: You know what, why don't we just enable math mode for ya?

Author: happy to change it if you do, but right now these are the only two exponents or complex math

Reviewer: ```suggestion
energies greater than ~50 TeV (5×10<sup>13</sup> eV) and issue alerts to the community.
```

> The IceCube realtime alert point of contact can be reached at roc@icecube.wisc.edu

Reviewer: ```suggestion
The IceCube realtime alert point of contact can be reached at roc@icecube.wisc.edu.
```

## GCN Notice Types in GCN Classic and GCN Classic Over Kafka

> IceCube has been sending alerts generated by realtime on astrophysical neutrino searches([1],[2])

Reviewer: It would be preferable for these references to be rendered like footnotes with hyperlinks. I will search for a Markdown plugin to do that.

Reviewer: We support GitHub Flavored Markdown, which [includes footnotes](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes). Would you please try the GitHub Flavored Markdown syntax?

Author: done, looks good, thanks for the pointer.

| `AMON_NU_EM_COINC`          | 4–8  | 0.2–1°     |

</div>

## References

[1] M.G. Aartsen, et al. (2017) Astropart. Phys., 92, 30 ([10.1016/j.astropartphys.2017.05.002](http://dx.doi.org/10.1016/j.astropartphys.2017.05.002))\
[2] R. Abbasi, et al. (2023) [https://arxiv.org/abs/2304.01174](https://arxiv.org/abs/2304.01174)\
[3] M. G. Aartsen et al. (2016) JINST 11 P11009 ([10.1088/1748-0221/11/11/P11009](https://doi.org/10.1088/1748-0221/11/11/P11009))\
[4] M. G. Aartsen et al. (2020) ApJL 898 L10 ([10.3847/2041-8213/ab9d24](https://doi.org/10.3847/2041-8213/ab9d24))\
[5] R. Abbasi et al. (2023) ApJ 944 80 ([10.3847/1538-4357/aca5fc](https://doi.org/10.3847/1538-4357/aca5fc))\
[6] I. Bartos et al. (2019) Phys. Rev. D 100, 083017 ([10.1103/PhysRevD.100.083017](https://doi.org/10.1103/PhysRevD.100.083017))

## IceCube Acknowledges

The IceCube Neutrino Observatory is funded and operated primarily through an award from
the National Science Foundation to the University of Wisconsin–Madison. The IceCube
Collaboration, with over 350 scientists in 58 institutions from around the world, runs an
extensive scientific program that has established the foundations of neutrino

> astronomy. See https://icecube.wisc.edu/collaboration/institutions.

Reviewer: ```suggestion
astronomy. See the [full list of institutions](https://icecube.wisc.edu/collaboration/institutions).
```

Track events can be reconstructed with an uncertainty of less than 1 degree, while cascade events
have higher signal purity. IceCube triggers on signals for neutrino energies greater than ~10 GeV
(10<sup>10</sup> eV) and can identify likely astrophysical neutrino events for neutrino

> energies greater than ~50 TeV (5x10<sup>13</sup> eV) and issue alerts to the community.

Reviewer: That's the Unicode multiplication sign glyph, not the letter "x."

```suggestion
energies greater than ~50 TeV (5×10<sup>13</sup> eV) and issue alerts to the community.
```

Author: Apologies, missed that
